docs/helm/README.md

```diff
@@ -99,6 +99,81 @@
   * **Code:** 403 BAD REQUEST <br />
     **Content:** `{ error : "error message" }`
 
+
+**Upgrade Helm Release**
+----
+  _Upgrade Helm release_
+
+* **URL**
+
+    `/api/helm/release`
+
+* **Method:**
+  
+  `PUT` 
+  
+*  **Put Data Params**
+
+```
+ {
+   name: [string],
+   namespace: [string]
+   chart_url: [string]
+   values: map[string]interface{}
+  }
+```
+
+*  **Example Request**
+```
+    {
+    	"name": "test-helm-release",
+    	"namespace": "default",
+    	"chart_url": "https://github.com/akashshinde/console/raw/helm_endpoints/pkg/helm/testdata/influxdb-3.0.2.tgz"
+        "values": { "service": {"type": "ClusterIP"} }
+    }
+```
+
+* **Success Response:**
+  
+  * **Code:** 200 <br />
+ 
+* **Error Response:**
+
+  * **Code:** 403 BAD REQUEST <br />
+    **Content:** `{ error : "error message" }`
+
+
+**Rollback Helm Release**
+----
+  _Rollback Helm release_
+
+* **URL**
+
+    `/api/helm/release`
+
+* **Method:**
+  
+  `PATCH` 
+  
+*  **Put Data Params**
+
+```
+ {
+   name: [string],
+   namespace: [string]
+   rollback_version: [int]
```

Comment:
I would rename it just to `version`, so it could match with `Release` object `version` field.

Reply:
Done